### ISSUE TITLE
VLC: update to 3.0.4

### ISSUE THREAD

**Issue by OpenSourceAnarchist** [clearlinux/distribution#173]
### What is the issue type?

* [ ] Bug report
* [X] Enhancement request

### Describe the problem

New official tarball is located here: [https://get.videolan.org/vlc/3.0.4/](https://get.videolan.org/vlc/3.0.4/)

VLC version in Clear Linux's repos is 3.0.3. I would issue a pull request myself with all the changes, but I have no idea what needs to be changed. I'd love to contribute to keep packages updated, but there is no such guide for Clear Linux and it is different than most other distros. Let me know if there is anything I can do in the future, or if submitting new issues is the preferred method.

**Comment by fenrus75** [comment]
our new version finder found this one already and I had updated this
morning.
Most of our package updates are automated (with us approving) so filing
requests like this is easiest... we can click the button for the automation
in case we hadn't already

**Comment by OpenSourceAnarchist** [comment]
Thank you for the clarification! The only reason I opened an issue was because the GitHub repo for clearlinux-pkgs/vlc did not show an update/commit yet. I guess that does not reflect all the work that is done on your end before a build. I can be more patient next time :P

On a side note, I made an issue request yesterday I believe about Cuppa, a version finder for distribution developers. If you already have a version finder, feel free to close the issue as it was merely a suggestion if you didn't have one yet.
